tests/resources/service-templates/tosca-simple-1.0/node-cellar/node-cellar.yaml

```diff
@@ -102,8 +114,10 @@ topology_template:
               #token: { token: [ 'zero.one|two-three', '.|-', 3 ] }
       interfaces:
         Maintenance:
-          enable: juju > charm.maintenance_on
-          disable: juju > charm.maintenance_off
+          inputs:
+            mode: eventual
+          enable: maintenance_node_cellar.sh
```

Comment:
I always want to keep the Node Cellar test resource as "rich" as possible in all features we support, as a sanity check. If we break something, Node Cellar will almost always fail.